agent/bootstrap.go

```diff
@@ -265,9 +265,8 @@ func acquireLock(path string, seconds int) (*lockfile.Lockfile, error) {
 	for true {
 		err := lock.TryLock()
 		if err != nil {
-			if err == lockfile.ErrBusy {
-				// Keey track of how many times we tried to get the
-				// lock
+			if te, ok := err.(interface{ Temporary() bool }); ok && te.Temporary() {
```

Comment:
Also `TemporaryError("Lockfile created, but doesn't exist")` is a weird error. I wonder why you'd get that?